indico/modules/events/templates/reviews/rating_widget.html

```diff
@@ -15,7 +15,7 @@
         </div>
         <div id="{{ field.id }}" class="rating-scale flexrow f-a-center">
             <span class="score">{{ '-' if field.object_data is none else field.data }}</span>
-            {% for bullet in range(cfa.rating_range[1], cfa.rating_range[0] - 1, -1) %}
+            {% for bullet in range(rating_range[1], rating_range[0] - 1, -1) %}
```

Comment:
Same place where the `cfa` was coming from :) Since I had to reuse the widget in the papers module, I couldn't pass the `cfa` object anymore.